recipes/_user.rb

```diff
@@ -35,13 +35,23 @@
   ignore_failure true if windows?
 end
 
+# Ensure '/usr/local/bin/bash' is added to acceptable shells list
+execute 'chsec_login_shell' do
+  command "sudo chsec -f /etc/security/login.cfg -s usw -a 'shells=/bin/sh,/bin/bsh,/bin/csh,/bin/ksh,/bin/tsh,/bin/ksh93,/usr/bin/sh,/usr/bin/bsh,/usr/bin/csh,/usr/bin/ksh,/usr/bin/tsh,/usr/bin/ksh93,/usr/bin/rksh,/usr/bin/rksh93,/usr/sbin/uucp/uucico,/usr/sbin/sliplogin,/usr/sbin/snappd,/usr/bin/bash,/opt/omnibus-toolchain/embedded/bin/bash'"
+  only_if { aix? }
+end
+
 user node['omnibus']['build_user'] do
   home     build_user_home
   supports manage_home: true
   password node['omnibus']['build_user_password']
   unless windows?
-    shell '/usr/local/bin/bash'
-    gid   node['omnibus']['build_user_group']
+    if aix?
+    shell node['omnibus']['build_user_shell']
+    else
+      shell '/usr/local/bin/bash'
```

Comment:
Works for me - that code felt really awkward and I couldn't think offhand of a better way.